plugin/advisor/pg/parser.go

```diff
@@ -9,6 +9,16 @@ import (
 func parseStatement(statement string) ([]ast.Node, []advisor.Advice) {
 	nodes, err := parser.Parse(parser.Postgres, parser.Context{}, statement)
 	if err != nil {
+		if _, ok := err.(*parser.ConvertError); ok {
+			return nil, []advisor.Advice{
+				{
+					Status:  advisor.Warn,
```

Comment:
Why we choose Warn instead of Error?

Reply:
The purpose of this PR is to differentiate between syntax errors and conversion errors. Specifically, I want conversion errors to not trigger syntax errors. So I chose the Warn level here.

But I now realize that the conversion error should also be at the Error level because it prevents the advisor from doing it properly. 

Now I have modified how syntax is implemented and use the Error level here. PTAL @tianzhou